frontend/src/pages/auth/components/facebook-login-button/facebook-login.tsx

```diff
@@ -0,0 +1,20 @@
+import { FacebookProvider } from 'react-facebook';
+
+import { config } from '~/libs/packages/config/config.js';
+import { type UserSignInWithFacebookResponseDto } from '~/packages/auth/auth.js';
+
+import { FacebookLoginButton } from '../components.js';
```

Comment:
But it looks like this button component is placed in same folder, are you sure you need to import it from `components` folder